languagetool-language-modules/ga/pom.xml

```diff
@@ -0,0 +1,85 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.languagetool</groupId>
+    <artifactId>languagetool-parent</artifactId>
+    <version>3.6-SNAPSHOT</version>
```

Comment:
I'd changed that but not bothered to commit. Until I can get the maven gpg plugin to sign while deploying to sonatype there are no dictionary binaries, so it's not usable.